src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/view/sw-first-run-wizard-introduction/sw-first-run-wizard-introduction.scss

```diff
@@ -0,0 +1,28 @@
+@import "~scss/variables";
+
+.sw-first-run-wizard-introduction {
+    width: 100%;
+    align-self: flex-start;
+    margin-top: 100px;
+
+    .center {
+        text-align: center;
+    }
+
+    .headline-welcome {
+        font-size: 32px;
+        font-weight: normal;
+        text-align: center;
+        letter-spacing: 0.2px;
+        line-height: 45px;
+    }
+
+    .text-welcome {
+        width: 675px;
```

Comment:
Oddly specific. Do other wizard pages do the same?

Reply:
i guess it can be removed